hack/local-setup.sh

```diff
@@ -191,21 +199,19 @@ deployDashboard $KIND_CLUSTER_CONTROL_PLANE 0
 #8. Add the control plane cluster
 argocdAddCluster ${KIND_CLUSTER_CONTROL_PLANE} ${KIND_CLUSTER_CONTROL_PLANE}
 
-
-
 #9. Add workload clusters if MCTC_WORKLOAD_CLUSTERS_COUNT environment variable is set
 if [[ -n "${MCTC_WORKLOAD_CLUSTERS_COUNT}" ]]; then
   for ((i = 1; i <= ${MCTC_WORKLOAD_CLUSTERS_COUNT}; i++)); do
     kindCreateCluster ${KIND_CLUSTER_WORKLOAD}-${i} $((${port80} + ${i})) $((${port443} + ${i}))
     installGatewayAPI ${KIND_CLUSTER_WORKLOAD}-${i}
     deployIngressController ${KIND_CLUSTER_WORKLOAD}-${i}
     deployIstio ${KIND_CLUSTER_WORKLOAD}-${i}
+    deployKuadrant ${KIND_CLUSTER_CONTROL_PLANE}-${i}
```

Comment:
Getting an error
```
error: no context exists with the name: "kind-mctc-control-plane-1"
make: *** [local-setup] Error 1
````

Should it be this?
```suggestion
    deployKuadrant ${KIND_CLUSTER_WORKLOAD}-${i}
```

Reply:
yes thank you copy paste error